docs/users-guide/13-embedding.md

```diff
@@ -0,0 +1,62 @@
+## Embedding Metabase in other Applications
+
+### Key Concepts
+
+#### Applications
+An important distinction to keep in mind is the difference between Metabase and the embedding application. The charts and dashboards you will be embedding live in the Metabase application, and will be embedded in your (i.e. the embedding application). 
```

Comment:
There's a missing word here: `and will be embedded in your`
I believe you want, `and will be embedded in your application (i.e., the embedding application).`